### PR TITLE
Exec to spawn

### PR DESCRIPTION
**Pull request by joshacheson**
WIP, Branched from https://github.com/danger/danger-js/pull/619

### CODE REVIEW

File: source/platforms/git/localGetCommits.ts [hunk 1]
```diff
@@ -14,30 +14,33 @@ const authorDate = "%ai"
 const committerName = "%cn"
 const committerEmail = "%ce"
 const committerDate = "%ci"
-const message = "%s" // this is subject, not message, so it'll only be one line
+const message = "%f" // this is subject, not message, so it'll only be one line
```
Comment (joshacheson) [inline source/platforms/git/localGetCommits.ts]: @orta this is the crux of it - curious if you're comfortable with this solution?

Relevant doc for reference: https://git-scm.com/docs/pretty-formats

Comment (orta) [reply source/platforms/git/localGetCommits.ts]: Yep, seems totally fine to me!

**Comment by orta** [comment]
Test fails are legit 👍

**Comment by joshacheson** [comment]
@orta fixed what I could, seems that the last remaining failure might be... not that legit?

**Comment by orta** [comment]
Agreed - rerunning that test, looks like doing `create-react-app init` failed which is definitely unrelated 👍

**Comment by orta** [comment]
merge on green

**Comment by orta** [comment]
OK, this got deployed in the latest release 👍

**Comment by joshacheson** [comment]
Thank you! Hope it solves the problem I was having :D

**Pull request merged**